### DISCUSSION TITLE
RetroScaler PS1 + 8BitDo P30 Board on SCPH-1080 digital only pad

### DISCUSSION THREAD

**Discussion by isermann** [darthcloud/BlueRetro#390] [Q&A] [answered]
Hello together,

I combined a RetroScaler BlueRetro PSX receiver with a 8BitDo PS1 P30 Bluetooth drop-in mod board.

Unfortunately, I cannot get the D-pad working.
Maybe this is related to the "analog" switch on the former Dual Shock controllers which is unavailable on the SCPH-1080 hence resulting in unavailable D-pad?

Looking forward to you reply's! Thanks!

**Comment by darthcloud** [comment, marked as answer]
probably because 8bitdo expose the dpad as a joystick

See https://github.com/darthcloud/BlueRetro/discussions/159 on how to use web config to map joystick to dpad.

**Reply by isermann** [reply]
Thank you!

**Reply by vicsidious** [reply]
Indeed, 8bitdo controllers often map dpad as left strick by default! 
I left a comment on topic #159 that might help you! 

Here it is: https://github.com/darthcloud/BlueRetro/discussions/159#discussioncomment-3871650

Have a good day! 
-David
